src/test/FuzzerImpl.cpp

```diff
@@ -27,12 +30,72 @@ namespace stellar
 namespace FuzzUtils
 {
 auto constexpr FUZZER_MAX_OPERATIONS = 5;
-auto constexpr INITIAL_LUMEN_AND_ASSET_BALANCE = 100000LL;
+auto constexpr INITIAL_ASSET_DISTRIBUTION = 100LL;
 auto constexpr NUMBER_OF_ASSETS_TO_ISSUE = 4;
+auto constexpr FUZZING_FEE = 1;
+auto constexpr FUZZING_RESERVE = 4;
+auto constexpr INITIAL_TRUST_LINE_LIMIT = 3 * INITIAL_ASSET_DISTRIBUTION;
+auto constexpr DEFAULT_NUM_TRANSACTIONS_TO_RESERVE_FEES_FOR = 10;
+auto constexpr DEFAULT_STROOPS_AVAILABLE_FOR_TEST_ACTIVITY = 32LL;
 
 // must be strictly less than 255
 uint8_t constexpr NUMBER_OF_PREGENERATED_ACCOUNTS = 16;
 
+class AccountParameters
```

Comment:
do we really need classes etc for this? Ideally all of this would be constexpr so that we don't have any overhead related to this at runtime

Reply:
The classes are for gathering the configuration parameter calculations into one place preceding all of the transaction application, the idea being to make it easy to see where to modify parameters, and dealing with cases where a single operation affects multiple parameters, such as how a single offer can induce reserve requirements and selling liabilities.

I will attempt to measure performance impacts.  But with classes or without, I'm not sure how we'd correctly pre-compute required balances as `constexpr`s: as @jonjove pointed out, we should be (and now are) using `getOfferSellingLiabilities()`, which would require changes outside the fuzzer to make usable as a `constexpr`.

I'm open to ideas about that, of course.

Reply:
(This has been changed in accordance with Jon's suggestion to start out with large balances and reduce them after the rest of setup.)